components/scream/src/share/io/scorpio_output.cpp

```diff
@@ -271,26 +279,30 @@ void AtmosphereOutput::register_dimensions(const std::string& name)
  *   field_repo: is a pointer to the field_repository for this simulation.
  *   name: is a string name of the variable who is to be added to the list of variables in this IO stream.
  */
+  using namespace scream;
```

Comment:
I think I made this change, looking at the pattern in other places in these changes... Should these two lines be replaced with `using namespace scorpio`?